en/tutorials-and-examples/cms/articles-controller.rst

```diff
@@ -1,9 +1,132 @@
 CMS Tutorial - Creating the Articles Controller
 ###############################################
 
-* Create the Controller
-* Add a index & view action.
-* Add the index & view template
+With our model created, we need a controller for our articles. Controllers in
+CakePHP handle HTTP requests and execute business logic contained in model
+methods, to prepare the response. We'll place this new controller in a file
+called **ArticlesController.php** inside the **src/Controller** directory.
+Here's what the basic controller should look like::
+
+    // src/Controller/ArticlesController.php
+
+    namespace App\Controller;
+
+    class ArticlesController extends AppController
+    {
+    }
+
+Now, let's add an action to our controller. Actions are controller methods that
+have routes connected to them. For example, when a user requests
+**www.example.com/articles/index** (which is also the same as
+**www.example.com/articles**), CakePHP will call the ``index`` method of your
+``ArticlesController``. This method should query the model layer, and prepare
+a response (via the View and Template). The code for that action would look like
+this::
+
+    // src/Controller/ArticlesController.php
+
+    namespace App\Controller;
+
+    class ArticlesController extends AppController
+    {
+
+        public function index()
+        {
+            $articles = $this->Articles->find('all');
+            $this->set(compact('articles'));
+        }
+    }
+
+By defining function ``index()`` in our ``ArticlesController``, users can now
+access the logic there by requesting **www.example.com/articles/index**. Similarly,
+if we were to define a function called ``foobar()``, users would be able to
+access that at **www.example.com/articles/foobar**.
+
+.. warning::
+
+    You may be tempted to name your controllers and actions in a way that allows
+    you to obtain specific URLs. Resist that temptation. Instead, follow the
+    :doc:`/intro/conventions` creating readable, meaningful action names. You
+    can then use doc:`/development/routing` to connect the URLs you want to the
+    actions you've created.
+
+Our controller action is very simple. It fetches all the articles from the
+database, using the model that is automatically loaded via naming conventions.
+It then uses ``set()`` to pass the articles into the view (which we'll create
```

Comment:
Here, "view" is in lowercase, [above](https://github.com/cakephp/docs/pull/4870/files#diff-51a543267d06defd6019191b3b648034R23) it's in uppercase. 😕